client/src/components/SearchBarTrace.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { JSXElementConstructor, useEffect } from 'react';
+import { changeView, selectSourceMap } from './sourceMapSlice';
+import { useAppSelector, useAppDispatch } from '../lib/hooks';
+import { selectTraceView } from './traceViewSlice';
+import { TraceData } from './trace-table/tableList';
+
+const SearchBarTrace = (): JSX.Element => {
+	
+
+  const traceViewData = useAppSelector(selectTraceView); 
+	const traceData: TraceData[] = [];
+
+	const dispatch = useAppDispatch();
+  const submitTrace = (traceID?:string | undefined):any => {
+    //updates the view data to reflect the new trace
+		console.log(traceID)
+		//dispatch(updateData({type: 1, data:traceID}))
```

Comment:
Is this purely for console logging, or are you dispatching the actual change? Dispatch is still commented out here.